### DISCUSSION TITLE
(solved) Battery voltage too high

### DISCUSSION THREAD

**Discussion by Basnl80** [universam1/iSpindel#596] [Q&A] [answered]
Hello,

My voltage is too high. The reading in my Ispindle is 5,34V. When I measure it it's 4,15V and that's too much for the conversation factor of 246,80. Is there anything what I can do?

**Comment by wilbrod** [comment, marked as answer]
Check the resistors on your iSpindel, they're either wrong size or need to
be swapped out.

**Reply by Basnl80** [reply]
Ik measure 4.7k ohm and 183k ohm. The 183 need to be 220k ohm right?

**Comment by wilbrod** [comment]
That's correct. Color code for it is : Red-red-black-orange and a 5th color
for tolerance.

Not sure if 183k would be sufficient or in which way it affects the
iSpindel but make sure they're soldered in the right spot and you didn't
mix up their spot on the board you're using.

**Reply by Basnl80** [reply]
The resistors are on the right place. Soo it's looks like the 184k ohm is the problem

**Reply by universam1** [reply]
yep, it should be 220k

**Comment by Basnl80** [comment]
Thanx everybody, the 220k resistor was the problem. The color code says 220k but after remove it it's measured 215k. If placed a new 220k and nu it's works great.
